src/scenic/domains/driving/behaviors.scenic

```diff
@@ -26,6 +26,28 @@ def setLaneFollowingPIDControllers(is_vehicle, dt):
 
     return lon_controller, lat_controller
 
+def setTurnPIDControllers(is_vehicle, dt):
```

Comment:
These new functions don't seem to get used. Is part of the PR missing?

Reply:
Indeed, there was one commit missing...

Reply:
Oops, now there are two copies of `setTurnPIDControllers` and `setLaneChangingPIDControllers`, and they have different numerical parameters so I'm not sure which one is correct. Can you please fix?